assets/inference/environments/mlflow-ubuntu20.04-py39-cpu-inference/context/conda_dependencies.yaml

```diff
@@ -3,7 +3,7 @@ channels:
 - conda-forge
 - anaconda
 dependencies:
-- python=3.9.13
+- python=3.12
```

Comment:
python version of environment should match the name.

Reply:
Thanks for the catch. This ia a new environment.